acceptance/openstack/fgs/v2/invocation_test.go

```diff
@@ -0,0 +1,63 @@
+package v2
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/fgs/v2/function"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/fgs/v2/invoke"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestFunctionGraphSync(t *testing.T) {
+	if funcGraph := os.Getenv("FUNCGRAPH_TEST"); funcGraph == "" {
```

Comment:
think you can run this test in zuul, no need to skip

Reply:
This test doesn't work because of some events in functiongraph that haven't figured out yet :)